site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -156,10 +156,12 @@ The list of roles that the User has for this registration.
 [field]#user.registrations``[x]``.timezone# [type]#[String]#::
 The User's preferred timezone for this registration. The string will be in an https://www.iana.org/time-zones[IANA] time zone format.
 
-[field]#user.registrations``[x]``.tokens# [type]#[Map<String,String>]# [since]#Available since 1.1.0#::
+[field]#user.registrations``[x]``.tokens# [type]#[Map<String,String>]# [deprecated]#Deprecated#::
 A map that contains tokens returned from identity providers.
 +
 For example, if this user has authenticated using the Facebook Identity Provider, the Facebook access token will be available in this map, keyed by name `Facebook`. For an OpenID Connect Identity provider, or other generic providers, if a token is stored it will be keyed by the Identity Provider unique Id.
++
+[deprecated]#Removed in version 1.28.0#
```

Comment:
We should add a link to the "Link API" so we can direct users to the new way to retrieve this token.

Reply:
I'll need @bguyza's Link API merged first.  Also, I think there are quite a few places that need changing for this, so maybe this should be a separate PR.

Reply:
Sounds good, perhaps we should open a separate issue so we don’t lose it.

Reply:
I moved this to #763.  I made the same deprecation in various places and removed `tokens` from the json where applicable.  It's in draft atm waiting for Brett's Link API to merge.